examples/basic/Count.js

```diff
@@ -1,4 +1,5 @@
-import React, { PropTypes } from 'react'
+import PropTypes from 'prop-types';
+import React from 'react';
```

Comment:
@davidascher CircleCI seems to be drunk again but could you please make sure this passes `npm run lint`? Our ESLint config doesn't allow semis.